### DISCUSSION TITLE
Github student pack

### DISCUSSION THREAD

**Discussion by BoumedieneSaidi** [community/community#144438] [GitHub Education]
### Hi everyone,

How long the github pack could be available ?

**Comment by mastertanzeel** [comment]
As long as you're a student.

**Comment by davevad93** [comment]
Hi @BoumedieneSaidi ,

GitHub Education benefits last for 2 years, after which you'll need to re-verify your student status if you want to renew them.

**Comment by lllHunterlllSV** [comment]
**Hello! @BoumedieneSaidi  👋**

Thank you for reaching out with this question!

The **GitHub Student Developer Pack** is typically available to students **for as long as they are actively enrolled in an accredited educational institution**. Here are some key details:

- 📅 **Duration**: Initially, the Pack is valid for **one year**. However, students can renew it annually as long as they can verify their student status. 
- 🎓 **Eligibility**: To renew, students need to confirm their eligibility with GitHub by providing proof of enrollment (like a student ID, transcript, or other documentation).
- 🔄 **Renewal Process**: GitHub usually sends an email reminder when it's time to renew, so keep an eye on your inbox! You can also check your [GitHub Education Dashboard](https://education.github.com/discount_requests) for renewal options.
- 💡 **Important Note**: Eligibility requirements may vary based on your location or institution, so it’s always a good idea to verify current guidelines from GitHub Education.
- 🔗 **More Info**: For the latest details on eligibility and renewals, check the [GitHub Education documentation](https://education.github.com/discount_requests/pack_application).

If you have any specific questions about your account or if your status has recently changed, contacting [GitHub Support](https://support.github.com/) can also be helpful. 😊 

Hope this helps, and best of luck with all your coding adventures!
